pdf/pdfannotation.cpp

```diff
@@ -97,7 +97,7 @@ void PDFAnnotation::attach(PDFDocument *document, PDFSelection *selection)
     
     QRectF bounds = word.second;
     for (int i = 1; i < selection->count(); i++)
-        bounds.united(selection->rectAt(i).second);
+        bounds = bounds.united(selection->rectAt(i).second);
```

Comment:
I think you can use |= like on line 336.

This `::attach()` is overloaded for the HighlightAnnotation, and in UI, only highlight annotations are calling the `::attach()` method. That's why this bug has been left unnoticed, I think.

Reply:
Oh, I'll switch to that for consistency.